yt/frontends/athena_pp/data_structures.py

```diff
@@ -153,13 +152,22 @@ def __init__(
         zrat = self._handle.attrs["RootGridX3"][2]
         self._nonuniform = xrat != 1.0 or yrat != 1.0 or zrat != 1.0
         self._magnetic_factor = get_magnetic_normalization(magnetic_normalization)
+
+        geom = self._handle.attrs["Coordinates"].decode("utf-8")
+        self.geometry = Geometry(geom_map[geom])
```

Comment:
This looks right to me!  The only other place that *might* need a check, I think, is where the default domain limits get set, to make sure they match the right order for pi and 2pi.  That may not be necessary if defaults are never required, however.